IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcRelInterferesElements/Documentation.md

```diff
@@ -1,12 +1,24 @@
-The _IfcRelInterferesElements_ objectified relationship indicates that two elements interfere. Interference is a spatial overlap between the two elements. It is a 1 to 1 relationship. The concept of two elements interfering physically or logically is described independently from the elements. The interference may be related to the shape representation of the entities by providing an interference geometry.  
+The _IfcRelInterferesElements_ objectified relationship indicates that two elements interfere. Interference is a spatial and/or shape interaction between the two elements. It is a 1 to 1 relationship. The concept of two elements interfering physically or logically is described independently from the elements. The interference may be related to the shape representation of the entities by providing an interference geometry.  
   
-* When the interference geometry is available it can be passed by the optional attribute _InterferenceGeometry_ pointing to _IfcConnectionGeometry_. The connection geometry is provided as a point, curve, surface, or solid within the placement coordinate systems of the connecting elements. The _IfcConnectionVolumeGeometry_ is the default type to be used for interference in 3D space, as indicated in e.g. clash detections.  
+* When the interference geometry is available it can be passed by the optional attribute _InterferenceGeometry_ pointing to _IfcConnectionGeometry_. The connection geometry is provided as a point, curve, surface, or volume within the local placement coordinate systems of the connecting elements. The _IfcConnectionVolumeGeometry_ is the default type to be used for interference in 3D space, as indicated in e.g. clash detections.  
```

Comment:
Volume is not a term in geometry items. Volume is a measurement type. Solid is the proper terminology.

Reply:
Not sure why I cannot add a suggestion here but this "coordinate systems of the connecting elements" is ambiguous. Which element? Only one local placement can serve as the basis. I guess Implied order should influence this also?

Reply:
```suggestion
* When the interference geometry is available it can be passed by the optional attribute _InterferenceGeometry_ pointing to _IfcConnectionGeometry_. The connection geometry is provided as a point, curve, surface, or solid within the local placement coordinate systems of the connecting elements. The _IfcConnectionVolumeGeometry_ is the default type to be used for interference in 3D space, as indicated in e.g. clash detections.  
```

Reply:
@SergejMuhic amusingly your are wrong and right at the same time! :D

1.  you are correct in the sense the term is solid, but amusingly the subtype is called _IfcConnectionVolumeGeometry_ so theres that !
2. it sounds ambiguous but is correct IfcConnectionGeometry subtypes are made up of 2 geometries one for the overlapping zone on _RelatingElement_ (in its placement) and the same for _RelatedElement_ (in its placement). so...! :p

Reply:
Solid was kept is the actual version

Reply:
> 
> 
> @SergejMuhic amusingly your are wrong and right at the same time! :D
> 
>     1. you are correct in the sense the term is solid, but amusingly the subtype is called _IfcConnectionVolumeGeometry_ so theres that !
> 
>     2. it sounds ambiguous but is correct IfcConnectionGeometry subtypes are made up of 2 geometries one for the overlapping zone on _RelatingElement_ (in its placement) and the same for _RelatedElement_ (in its placement). so...! :p

You have to read my comments carefully. And I would appreciate that you refrain from labeling them wrong. Thank you.

> Volume is not a term in geometry items.

That is a fact. IfcConnectionGeometry is by no means a geometric item. It has been introduced specifically to not be a geometric item (or a representation item) but to "describe the geometric and topological constraints that facilitate the physical connection of two objects". Semantics.